process/dataindexer/workItems/workItemBlock.go

```diff
@@ -39,13 +39,17 @@ func (wib *itemBlock) Save() error {
 		return nil
 	}
 
-	defer func(startTime time.Time, headerHash []byte, headerNonce uint64) {
+	headerNonce := wib.outportBlockWithHeader.Header.GetNonce()
+	headerHash := wib.outportBlockWithHeader.BlockData.HeaderHash
+	shardID := wib.outportBlockWithHeader.Header.GetShardID()
+	defer func(startTime time.Time, headerHash []byte, headerNonce uint64, shardID uint32) {
```

Comment:
You can simplify this call by simply calling the defer log.Debug....

Reply:
refactored.